RecoMuon/GlobalTrackingTools/interface/ThrParameters.h

```diff
@@ -1,3 +1,6 @@
+#ifndef ThrParameters_h
+#define ThrParameters_h
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h

Reply:
ping